designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiErrorMessages.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.ui.api
+
+object TestingApiErrorMessages {
+
+  val noSourcesWithTestDataGeneration = "No sources with test data generation available"
+
+  val noDataGenerated = "No data was generated"
+
+  val noInputRecords = "No input records found"
+
+  def requestedTooManySamplesToGenerate(maxSamples: Int) =
+    s"Too many samples requested, limit is $maxSamples. Please configure 'testDataSettings.maxSamplesCount' to increase the limit"
+
+  def tooManyCharactersGenerated(length: Int, limit: Int) =
+    s"$length characters were generated, limit is $limit. Please configure 'testDataSettings.testDataMaxLength' to increase the limit"
```

Comment:
I missed the most important information - "Too many characters generated". Let's start the sentence with this. The same in two messages below. 
Also, the same as with `noDataGenerated`, the obvious question is why you, machine, generated too many characters? Did you have some internal problem and generate them in some infinite loop? ;)  Better message would be for example:
"Too many characters were found in the sample of data when trying to provide $maxSample sample"